src/components/UI/DataTable.tsx

```diff
@@ -1,8 +1,11 @@
-import { Table, Thead, Tr, Th, TableContainer, Text, Tbody, Td } from '@chakra-ui/react';
+import { Link, Table, Thead, Tr, Th, TableContainer, Text, Tbody, Td } from '@chakra-ui/react';
 import { FC } from 'react';
+import { ReleaseData } from '../../types';
+import { getParsedDate } from '../../utils';
 
 interface Props {
   columnHeaders: string[];
+  // TODO: update data type
   data: any;
```

Comment:
```suggestion
  data: ReleaseData[];
```

Just looking through code, I think we can add this typing for the TODO

Reply:
@corwintines not possible for now as `DataTable` is also being used for OpenPGP Signatures table, where data type is different

Reply:
Sounds good, will approve then